README.md

```diff
@@ -1,2 +1,222 @@
 # rweather-decoder
-Decoders of various weather reports
+
+Decoders of various weather reports.
+
+The decoders are binary applications storing decoded reports in JSON files which are suitable for further machine processing.
+
+## Roadmap
+
+* [x] METAR / SPECI
+  * [ ] wind shear
+  * [ ] sea state
+  * [ ] TREND forecast
+  * [ ] REMARK section
+* [ ] TAF
+* [ ] SYNOP
+
+## Installation
+
+Install the decoders by running:
+
+```
+cargo install rweather-decoder
+```
+
+## Usage
+
+```
+[filip@fractal ~]$ decode-metar -h
+rweather-decoder 0.1.1
+CLI decoder of METAR files
+
+USAGE:
+    decode-metar [FLAGS] [OPTIONS] <input-globs>... <output>
+
+FLAGS:
+    -h, --help            Prints help information
+    -p, --pretty-print    Enable pretty-printing of output JSON file
+    -q, --quiet           Quiet
+    -V, --version         Prints version information
+
+OPTIONS:
+    -a, --anchor-time <anchor-time>    Anchor time (YYYY-MM-DD) for the plain file format. Specifies a day close to the
+                                       one when the reports were collected. If given, the individual METAR day will be
+                                       matched against it to create a proper datetime representation
+    -f, --file-format <file-format>    METAR file format (noaa-metar-cycles, plain) [default: noaa-metar-cycles]
+
+ARGS:
+    <input-globs>...    Input files (glob patterns separated by space)
+    <output>            Output JSON file. Same input reports will be deduplicated
+```
+
+## Example
+
+Check for latest METAR reports at https://tgftp.nws.noaa.gov/data/observations/metar/cycles/.
+
+We will download the file `16Z.TXT` (cycle 16Z) and run the `decode-metar` tool as follows:
+
+```
+decode-metar 16Z.TXT 16Z.json -p
```

Comment:
Based on the USAGE section, this would be better to read as: `decode-metar -p 16Z.TXT 16Z.json`

Reply:
Fixed